packages/charting/src/components/PieChart/PieChart.types.ts

```diff
@@ -0,0 +1,99 @@
+import { ITheme, IStyle } from '../../Styling';
+import { IStyleFunctionOrObject } from '../../Utilities';
+
+export interface IPieChart {}
+
+export interface IDataPoint {
+  /**
+   * Independent value of the data point, rendered.
+   */
+  value: number;
+
+  /**
+   * Dependent value of the data point, rendered.
+   */
+  label: number | string;
+}
+
+export interface IPieChartProps {
+  /**
+   * Data to render in the chart.
+   */
+  data?: IDataPoint[];
+
+  /**
+   * Width of the chart.
+   */
+  width?: number;
+
+  /**
+   * Height of the chart.
+   */
+  height?: number;
+
+  /**
+   * colors to render in the chart.
+   */
+  colors?: string[];
+
+  /**
+   * Title to apply to the whole chart.
+   */
+  chartTitle?: string;
+
+  /**
+   * Additional CSS class(es) to apply to the PieChart.
+   */
+  className?: string;
+
+  /**
+   * Theme (provided through customization.)
+   */
+  theme?: ITheme;
+
+  /**
+   * Call to provide customized styling that will layer on top of the variant rules.
+   */
+  styles?: IStyleFunctionOrObject<IPieChartStyleProps, IPieChartStyles>;
+
+  /**
+   * Width of line stroke
+   */
+  strokeWidth?: number;
+}
+
+export interface IPieChartStyleProps {
+  /**
+   * Theme (provided through customization.)
+   */
+  theme: ITheme;
+  /**
+   * Additional CSS class(es) to apply to the PieChart.
+   */
+  className?: string;
+  /**
+   * Width of the chart.
+   */
+  width: number;
+  /**
+   * Height of the chart.
+   */
+  height: number;
+}
```

Comment:
We recommend a typing that makes use of the component's props. So here, for example, it would be: 

```
export type IPieChartStyleProps =
  Required<Pick<IPieChartProps, 'theme' | 'width' | 'height'>> &
  Pick<IPieChartProps, 'className'>;
```

I tested this for you on your branch to make sure it works. 😄 

The `Required` line lists properties that must be provided while the next line lists optional props. You can also add props here that are not in `IPieChart` props if you need to. You can see `TeachingBubble.types.ts` for an example of that, if needed.

The benefit of this approach is that it binds `IPieChartProps` and `IPieChartStyleProps` interfaces together, which means if one changes, the other should also change and will be reflected with typing errors.